packages/flutter/lib/src/material/text_field.dart

```diff
@@ -813,19 +813,20 @@ class TextField extends StatefulWidget {
     BuildContext context,
     EditableTextState editableTextState,
   ) {
-    final Offset anchor =
-      SpellCheckSuggestionsToolbar.getToolbarAnchor(editableTextState.contextMenuAnchors);
-    final List<ContextMenuButtonItem>? buttonItems =
-      SpellCheckSuggestionsToolbar.buildButtonItems(editableTextState);
-
-    if (buttonItems == null){
-      return const SizedBox.shrink();
+    switch (defaultTargetPlatform) {
```

Comment:
Can you update the documentation on this method to explain the adaptiveness of the toolbar implemented here?

Reply:
Yes, good catch, it's out of date now.